services/templates.go

```diff
@@ -0,0 +1,213 @@
+//
+// Copyright (c) 2018, Joyent, Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+
+package services
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"path"
+
```

Comment:
struct field AccountId should be AccountID

Reply:
Yeah I know, we need to change `AccountId` within [joyent/triton-service-groups](http://github.com/joyent/triton-service-groups) first.

Reply:
Actually, with joyent/triton-service-groups#13 merged I can just use struct tags for the JSON parsing and rename to `AccountID` just on the `triton-go` side.

@stack72 opened joyent/triton-service-groups#14 for the renaming.